gossip/gossip.go

```diff
@@ -63,6 +63,9 @@ func (g *GossipMemberSet) Start(h pilosa.BroadcastHandler) error {
 
 // Seeds returns the gossipSeeds determined by the config.
```

Comment:
Might be good to clarify the function comment to something like: "Seeds returns a list of usable gossip seeds for this cluster. If no seeds are configured, return the gossip listener of this server."

Or otherwise comment on the `if` statement to explain it.

Reply:
Thanks @codysoyland, that was definitely confusing. I pushed a commit which renames that method and doesn't use `gossipSeeds` at all. It's really only there for one test. Ideally we could get ride of that logic completely, but for now I just tried to make it more clear what was happening.

Reply:
The latest commit is great 👍